recipes/monovar/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "MonoVar" %}
+# MonoVar doesn't have versions, so I'm just freezing the commit 7b47571d85a377c425feea86f6e591ee832fe342 as 'bioconda'
+{% set version = "bioconda" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    url: https://bitbucket.org/dlaehnemann/monovar/get/{{ version }}.tar.gz
+    md5: 90b338e4f4021f9808f4fbc222adc2ed
+
+build:
+    script: cp -r ${SRC_DIR}/src/ $PREFIX/lib/monovar && ln -s $PREFIX/lib/monovar/monovar.py $PREFIX/bin/monovar
+    number: 0
+    skip: True # [not py27]
+
+requirements:
+    build:
+        - python
+        - numpy
+        - scipy
+        - pysam ==0.8.3
+
+    run:
+        - python
+        - numpy
+        - scipy
+        - pysam ==0.8.3
+        - samtools ==0.1.19
+
+# MonoVar does not provide the ref.fa used to generate the provided example test bam files, so testing files from samtools were included in the bioconda repo version
+test:
+    source_files:
+        - examples/filenames.txt
+        - examples/mpileup.ref.fa
+        - examples/mpileup.ref.fa.fai
+        - examples/data_files/mpileup.1.bam
+        - examples/data_files/mpileup.1.bam.bai
+        - examples/data_files/mpileup.2.bam
+        - examples/data_files/mpileup.2.bam.bai
+        - examples/data_files/mpileup.3.bam
+        - examples/data_files/mpileup.3.bam.bai
+    commands:
+        - samtools mpileup -BQ0 -d10000 -f examples/mpileup.ref.fa -q 40 -b examples/filenames.txt | monovar -p 0.002 -a 0.2 -t 0.05 -m 2 -f examples/mpileup.ref.fa -b examples/filenames.txt -o output.vcf
```

Comment:
I can try. This would also get around the `mulled-build` failing due to the tests on actual files that don't seem to end up anywhere in the docker container -- or at least not where the test command expects them. Or am I missing something about setting up tests on actual `source_files` for the docker container?